.travis.yml

```diff
@@ -1,4 +1,6 @@
-language: generic
+language: node_js
+node_js:
+  - "10"
```

Comment:
What is using this?

Reply:
Installing `esy` requires a node environment, which we can either manage ourselves or let Travis manage for us.